src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { PreparedTodo } from './types/PreparedTodo';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoAddForm } from './components/TodoAddForm';
+import { findUserById } from './utils';
+
+const preparedTodos: PreparedTodo[] = todosFromServer
+  .map((todo: Todo) => {
+    return {
+      ...todo,
+      user: findUserById(todo.userId, usersFromServer),
+    };
+  });
 
 export const App = () => {
+  const [todos, setTodos] = useState(preparedTodos);
+
+  const handleAddTodo = (todo: PreparedTodo): void => {
+    setTodos((prevTodos) => [...prevTodos, todo]);
+  };
+
   return (
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/todos" method="POST">
-        <div className="field">
-          <input type="text" data-cy="titleInput" />
-          <span className="error">Please enter a title</span>
-        </div>
-
-        <div className="field">
-          <select data-cy="userSelect">
-            <option value="0" disabled>Choose a user</option>
-          </select>
-
-          <span className="error">Please choose a user</span>
-        </div>
-
-        <button type="submit" data-cy="submitButton">
-          Add
-        </button>
-      </form>
-
-      <section className="TodoList">
-        <article data-id="1" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">
-            delectus aut autem
-          </h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="15" className="TodoInfo TodoInfo--completed">
-          <h2 className="TodoInfo__title">delectus aut autem</h2>
-
-          <a className="UserInfo" href="mailto:Sincere@april.biz">
-            Leanne Graham
-          </a>
-        </article>
-
-        <article data-id="2" className="TodoInfo">
-          <h2 className="TodoInfo__title">
-            quis ut nam facilis et officia qui
-          </h2>
+      <TodoAddForm
+        todos={todos}
+        users={usersFromServer}
```

Comment:
Can we just import these `usersFromServer` inside component instead of passing them through props